CityOfDetroit/COD-Design-System
# Import components top down instead of bottom up

# Fix #134 

See root cause of the issue and possible solutions in the issue #134 

This PR:
* Import outermost components (organisms) first, then inner components (molecules, atoms) to ensure outer elements are defined before inner elements just as they are typically parsed in the DOM.
* Rebuild the production build files

As mentioned in issue #134 , a better (though more involved fix) will be to implement `table-stacked` as a reactive property that works regardless of the order of component definition, although I plan to do that as a followup since it will require a bit more time. This PR unblocks use of the table elements in production.

## Testing

1. Checkout demo branch `demo.issue.134`
2. Load the demo HTML page that defers script execution: `demo/demo-page-defer.html`
3. Ensure that `table-stacked` class is present on the table row in `cod-table-row` shadow DOM

![Screenshot 2023-12-28 at 1 39 42 PM](https://github.com/CityOfDetroit/COD-Design-System/assets/143553259/09182da9-3da4-4df5-bb62-e9371276b77f)
